### ISSUE TITLE
One of the firewall rules fail to start

### ISSUE THREAD

**Issue by lwchkg** [arno-iptables-firewall/aif#43]
Here is the dump:
```
[root@rees log]# service arno-iptables-firewall start
Starting Arno's Iptables Firewall...
/sbin/iptables -A HOST_BLOCK_DST_DROP -m limit --limit 1/m --limit-burst 1 -j LOG --log-level info --log-prefix AIF:Blocked host(s) Outbound:
ERROR (2): iptables v1.4.7: Maximum prefix length 29 for --log-prefix
Try `iptables -h' or 'iptables --help' for more information.
Nov 23 17:26:11 WARNING: 1 firewall rules failed to apply!
```

**Comment by arnova** [comment]
Thanks for reporting this. It has already been fixed in version 2.0.2a. So simply upgrade to this version and your issue should be resolved.

**Comment by lwchkg** [comment]
Thanks! Didn't note that there is a old version in the last hour!
